src/client/entity-editor/author-section/author-section.tsx

```diff
@@ -322,7 +322,7 @@ function mapDispatchToProps(dispatch: Dispatch<Action>): DispatchProps {
 		onEndDateChange: (endDate) =>
 			dispatch(debouncedUpdateEndDate(endDate)),
 		onEndedChange: (event) =>
-			dispatch(updateEnded(event.target.checked)),
+			dispatch(updateEnded((event.target as any).checked)),
```

Comment:
The reason I have `any` here is to mark it as something to come back to. Ideally we wouldn't need to do a type assertion here, because the `event` would have the correct type from the `onEndedChange` definition in `DispatchProps`. We could change it to HTMLInputElement, but I'm worried about it getting forgotten about (eslint will highlight any use of `any`).